cocos/native-binding/impl.ts

```diff
@@ -124,6 +124,20 @@ if( NATIVE ){
             globalJsb.__JsbBridgeWrapper = value;
         },
     });
+    const originSaveImageData = globalJsb.saveImageData;
+    globalJsb.saveImageData = (data: Uint8Array, width: number, height: number, filePath: string,
+        callback: (isSuccess: boolean) => void) => {
```

Comment:
`callback` should be optional

Reply:
I finally decide to remove the callback as the promise is sufficient.